core/src/prediction/InstrumentalPredictionStrategy.cpp

```diff
@@ -243,6 +243,6 @@ std::vector<double> InstrumentalPredictionStrategy::compute_debiased_error(
     error_bias += (residual_loto - residual) * (residual_loto - residual);
   }
   
-  error_bias *= ((num_trees - 1) / num_trees);
+  error_bias *= ((double)(num_trees - 1)) / (double)(num_trees);
```

Comment:
I would just have one cast (two is redundant).

Reply:
Done.